server/src/main/java/org/apache/iotdb/db/engine/flush/pool/AbstractPoolManager.java

```diff
@@ -84,4 +84,12 @@ public int getCorePoolSize() {
   public abstract void stop();
 
   public abstract String getName();
+
+  public int getNumberOfWorkingTasks() {
+    return ((ThreadPoolExecutor)pool).getActiveCount();
+  }
+
+  public int getNumberOfPendingTasks() {
+    return ((ThreadPoolExecutor)pool).getQueue().size();
+  }
```

Comment:
Actually , the method has the same function with `getWaitingTasksNumber` in line 68 of this class.

Reply:
fixed